src/Tpay/Provider/TpayApiChannelListProviderInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommerceWeavers\SyliusTpayPlugin\Tpay\Provider;
+
+interface TpayApiChannelListProviderInterface
+{
+    public function provide(): array;
```

Comment:
It would be nice to add an array type here.

Don't know what the array contains, but for example:
```suggestion
    /** @return array{id: int} */
    public function provide(): array;
```